### PR TITLE
A tiny cosmetic change to rpc/send.go: Use log.V(1).Infof()

### PR DESCRIPTION
**Pull request by kkaneda**
This is slightly simpler than if `log.V(1) { log.Info(...) }`.

### CODE REVIEW

**Comment by tschottdorf** [comment]
LGTM

File: rpc/send.go [hunk 1]
```diff
@@ -151,9 +151,7 @@ func Send(opts Options, method string, addrs []net.Addr, getArgs func(addr net.A
 				continue
 			}
 			reply := getReply()
-			if log.V(1) {
```
Comment (spencerkimball) [inline rpc/send.go]: This isn't just a cosmetic change, btw. These two formulations do something differently. In the original, only the log level is consulted. In the second, clients[index].Addr() is invoked. Not sure if it matters here, but something to keep in mind. Using:

    if log.V(1) {
      ...
    }

Can be a lot more efficient at times.

**Comment by spencerkimball** [comment]
LGTM

File: rpc/send.go (hunk 1 above)
Comment (kkaneda) [inline rpc/send.go]: Ah, got it. I didn't realize that point. Thanks, @spencerkimball !

**Pull request merged**